app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -115,6 +115,7 @@ def form_class
 
     def build_form
       @form = form_class.new(curation_concern, current_ability)
+      @collections = Collection.all
```

Comment:
The return value of this method has changed, which may or may not be an issue. I'm also concerned about adding another instance variable for the view to use.

Suggestion:

1. Add form_class#collections_for_select method that can be referenced in the view